config/initializers/session_store.rb

```diff
@@ -1,8 +0,0 @@
-# Be sure to restart your server when you modify this file.
-
-SharedLists::Application.config.session_store :cookie_store, key: '_sharedLists_session', path: ENV['RAILS_RELATIVE_URL_ROOT'] || '/'
```

Comment:
Do we still support running sharedlists in a subpath? I think that could be important for certain hosters.

Reply:
There is no difference to the old [setting](https://github.com/foodcoops/sharedlists/blob/master/config/initializers/session_store.rb). According to the [documentation](https://guides.rubyonrails.org/configuring.html#deploy-to-a-subdirectory-relative-url-root) one can use `config.relative_url_root` as an option and set `RAILS_RELATIVE_URL_ROOT`. So I think it's up to the hoster to set a subpath, isn't it?

Reply:
Indeed, it would work.
This was here for security, to make sure the sharedlists cookie wasn't shared with other apps running on the same domain.
Perhaps we could leave this initializer (or perhaps it needs adaptation for Rails 6)?

Reply:
Done. I added the missing file and added the secure flag.